frontend/packages/ceph-storage-plugin/src/components/odf-system/actions.tsx

```diff
@@ -18,8 +18,8 @@ export const getGenericActions = () => [...Kebab.factory.common];
 
 export const getActionsForOCS = () => [addStorage, ...getGenericActions()];
 
-export const getActions = (kind: string) => {
-  if (referenceForModel(OCSServiceModel).toLowerCase() === kind.toLowerCase())
+export const getActions = (kind: string, onlyMCG: boolean) => {
+  if (referenceForModel(OCSServiceModel).toLowerCase() === kind.toLowerCase() && !onlyMCG)
```

Comment:
Don't we need to block for `OCS_INDEPENDENT_FLAG` here as well ?

Reply:
true ! completely forgot about this